c.html.markdown

```diff
@@ -306,7 +307,25 @@ int main(void) {
     exit(-1);
     break;
   }
-
+  
+  // using "goto"
+  typedef enum { false, true } bool;
+  // for C don't have bool as data type :(
+  bool disaster = false;
+  int i, j;
+  for(i=0;i<100;++i)
```

Comment:
No, you are not mistaken. I don't know why I thought it was not possible, it is just nested loops not that weird. I had just never seen it done like this, looked weird or something.